GridTemp.ts

```diff
@@ -0,0 +1,44 @@
+/**  Class to manage the grid template and display records*/
+class GridTemplate {
+	private columnNames: ColumnName[] = [];
+	private dataRecords: GridData[] = [];
+
+	/** Initializes the column names and data records that will be used to display records in the grid. */
+	constructor(columnNames: ColumnName[], dataRecords: GridData[]) {
+		this.columnNames = columnNames;
+		this.dataRecords = dataRecords;
+	}
+
+	/*Display records in a grid in table format*/
```

Comment:
You are missing one `*` at the beginning of your javadoc comment, and please add spaces after the `/**` and before the `*/`